### DISCUSSION TITLE
Does libMesh support cohesive element?

### DISCUSSION THREAD

**Discussion by Traiwit** [libMesh/libmesh#3042] [Q&A] [answered]
Does libMesh support cohesive elements? 

if not, is it possible to create one from pre-existing elements that libmesh has?

Kind regards,
Traiwit

**Comment by jwpeterson** [comment, marked as answer]
I think cohesive elements have been done in the [MOOSE framework](https://www.github.com/idaholab/moose) (a FE framework which is based on libmesh) but I'm not totally sure... You may want to make a new Discussion or search past Discussions there on this topic.
